### DISCUSSION TITLE
Safe to use on scale?

### DISCUSSION THREAD

**Discussion by nerrixde** [agenda/agenda#1217] [Q&A] [answered]
agenda is literally **exactly** what I need.
Found this [remark](https://jobscheduler.net/#/?id=alternatives-that-are-not-production-ready)
![image](https://user-images.githubusercontent.com/30556676/106277521-8c36ed80-6239-11eb-9566-b703a7907a8e.png)

so I am unsure if agenda is safe to use in production, on scale.
Did the situation change since when this comment was made or what is to think about this remark?

**Comment by koresar** [comment, marked as answer]
Yes, the situation did change.
AFAIK memory leaks were fixed in the latest release(s). I can't guarantee there are no leaks, although people stopped reporting memory leaks.

We are using it in production. 

What scale are you looking at?

I know @simllll is using it at rather large scale. Maybe he can give you better outlook.

**Reply by nerrixde** [reply]
Thank you very much, nice to hear. 
I already started implementing it into my stack, really great.
For my scale I am confident it will work, as I am not speaking about Twitter-scale but more like startup-scale. 
Tysm for your awesome work :3

**Comment by ichie-Ozor** [comment]
@koresar I need urgent help
I built an app for a client that was supposed to send SMS to his customers on their birthdays and wedding anniversaries. I finished the job, and it's working fine. Still, when I hoisted it on Heroku server, it showed that SMS were sent, but the client did not receive the SMS while I received it. Both of us will receive the SMS offline, but I am the only one receiving it online, while he is not. What can I do, please? I need urgent help on this
